client/app/assets/less/inc/list-group.less

```diff
@@ -17,7 +17,12 @@
     }
 }
 
- .list-group-item {
+.max-character {
+  text-overflow: ellipsis;
+  overflow: hidden;
+}

```

Comment:
It also needs `white-space: nowrap` rule; even better is to replace all that with macro from Bootstrap: `.text-overflow();`